setup.py

```diff
@@ -21,6 +21,7 @@
 import setuptools
 from setuptools.command import build_ext
 from distutils import spawn
+from distutils import sysconfig
```

Comment:
Why distutils instead of Python 3.7s sysconfig?
```suggestion
import sysconfig
```

Reply:
I don't want to require Python 3.7, `distutils` in setup.py seems less bad than in general Python and we are using it for `spawn` already.